framework/test/unit/modules/pos/module.spec.ts

```diff
@@ -276,13 +276,6 @@ describe('PoS module', () => {
 				);
 			});
 
-			it('should fail if registerValidatorKeys return false', async () => {
-				(pos['_validatorsMethod'].registerValidatorKeys as jest.Mock).mockResolvedValue(false);
-
-				await expect(pos.initGenesisState(context)).toResolve();
-				await expect(pos.finalizeGenesisState(context)).rejects.toThrow('Invalid validator key');
-			});
```

Comment:
This test case belongs to `initGenesisState` test suite. Instead of removing it, it would be nice to adapt it into `should fail if registerValidatorKeys fails`, so we don't lose coverage of `initGenesisState`.